stdlib/ballerina-builtin/src/main/ballerina/ballerina/compression/natives.bal

```diff
@@ -0,0 +1,22 @@
+package ballerina.compression;
+
+@Description { value:"Function to decompress compressed bytes"}
+@Param {value:"content: Byte array of the compressed file"}
```

Comment:
Reformat the file. Different conventions used with whitespaces

Reply:
Fixed with f97b66c